addons/io_scene_gltf2_msfs/blender/msfs_material_function.py

```diff
@@ -773,13 +773,15 @@ def setDetailCompTex(self, tex):
 
     def setRoughnessScale(self, scale):
         nodeRoughnessScale = self.getNodeByName(MSFS_ShaderNodes.roughnessScale.value)
-        nodeRoughnessScale.outputs[0].default_value = scale
-        self.updateCompLinks()
+        if nodeRoughnessScale is not None:
```

Comment:
I don't think these checks are needed

Reply:
The Sub-version - I believe is a good idea, or change the version to 1.3.4 - so people know when a new version is out,

Reply:
I noticed that if users bring in models from say a third party site (CGTrader) or from the legacy exporter - that there could be differences where these nodes are not there and then an error happens without a check and the node build fails.  It would be helpful to you where devs post an issue they can't resolve.